package.json

```diff
@@ -35,18 +35,35 @@
     ]
   },
   "dependencies": {
-    "check-disk-space": "2.1.0"
+    "@grpc/proto-loader": "^0.5.3",
+    "@nestjs/platform-express": "^7.0.5",
+    "@nestjs/platform-fastify": "^7.0.5",
+    "amqp-connection-manager": "^3.2.0",
+    "amqplib": "^0.5.5",
+    "check-disk-space": "2.1.0",
+    "deprecate": "^1.1.1",
+    "grpc": "^1.24.2",
+    "mongoose": "^5.9.5",
+    "mqtt": "^3.0.0",
+    "nats": "^1.2.4",
+    "redis": "^3.0.2",
```

Comment:
Why these are defined as `dependencies`? Do I have to install all the packages? Note `grpc` requires some C++ libs, it would make a significant impact

Reply:
Oups, not supposed to be here. Terminus only has 2 direct `dependencies` (`deprecate` and `check-disk-space`), 4 `peerDependencies` and the rest is loaded dynamically.
Is removed